src/Jackett.Common/Indexers/Newpct.cs

```diff
@@ -418,55 +301,39 @@ private async Task<IEnumerable<ReleaseInfo>> TvSearch(Uri siteLink, TorznabQuery
 
         private async Task<IEnumerable<ReleaseInfo>> GetReleasesFromUri(Uri uri, string seriesName)
         {
-            var newpctReleases = new List<ReleaseInfo>();
-            var results = await RequestStringWithCookiesAndRetry(uri.AbsoluteUri);
-            await FollowIfRedirect(results);
+            var releases = new List<ReleaseInfo>();
 
-            //Episodes list
+            // Episodes list
+            var results = await RequestStringWithCookiesAndRetry(uri.AbsoluteUri);
             var seriesEpisodesUrl = ParseSeriesListContent(results.Content, seriesName);
+
+            // TV serie list
             if (!string.IsNullOrEmpty(seriesEpisodesUrl))
             {
-                var pg = 1;
-                while (pg < _maxEpisodesListPages)
-                {
-                    var episodesListUrl = new Uri(string.Format(_seriesUrl, seriesEpisodesUrl, pg));
-                    results = await RequestStringWithCookiesAndRetry(episodesListUrl.AbsoluteUri);
-                    await FollowIfRedirect(results);
-
-                    var items = ParseEpisodesListContent(results.Content);
-                    if (items == null || !items.Any())
-                        break;
-
-                    newpctReleases.AddRange(items);
-
-                    pg++;
-                }
+                results = await RequestStringWithCookiesAndRetry(seriesEpisodesUrl);
+                var items = ParseEpisodesListContent(results.Content);
+                if (items != null && items.Any())
+                     releases.AddRange(items);
             }
-            return newpctReleases;
+            return releases;
         }
 
-        private IEnumerable<Uri> SeriesListUris(Uri siteLink, string seriesName)
+        private IEnumerable<Uri> SeriesListUris(string seriesName)
         {
             IEnumerable<string> lettersUrl;
             if (!_includeVo)
-            {
                 lettersUrl = _seriesLetterUrls;
-            }
             else
-            {
-                lettersUrl = _seriesLetterUrls.Concat(_seriesVOLetterUrls);
-            }
+                lettersUrl = _seriesLetterUrls.Concat(_seriesVoLetterUrls);
```

Comment:
Would we want to use Union instead (same as `Concat(x).Distinct()`) to avoid duplicate links?